src/components/articleSummary/Index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Card,
+  Row,
+  Col
+} from 'react-bootstrap';
+import './articleSummary.scss';
+import PropTypes from 'prop-types';
+
+const ArticleSummary = (props) => {
+  const {
+    src, header, name, time
+  } = props;
+  const date = new Date(time).getTime();
```

Comment:
@olajide1234 I noticed this in Jacinta's PR. If this is a logic that will be repeated throughout the app, we should extract it into a function.